app/providers/gcp.py

```
@@ -1,24 +1,107 @@
         """Return the name of the cloud provider."""
         return "gcp"
 
     def get_iam_policies(self) -> Dict[str, Any]:
         """Retrieve IAM policies from GCP."""
-        # Mock implementation for now
+        if self.use_mock or not self._iam_service:
+            return self._get_mock_iam_policies()
+        
+        try:
+            from google.api_core.exceptions import GoogleAPIError
+            from google.iam.v1 import policy_pb2
```

Ruff (0.11.9)

`google.iam.v1.policy_pb2` imported but unused

Remove unused import: `google.iam.v1.policy_pb2`

(F401)